### DISCUSSION TITLE
RadiusNeighborsClassifier Support

### DISCUSSION THREAD

**Discussion by SebNav** [intel/scikit-learn-intelex#1583] [Ideas]
Hello, I like this project and regularly use several of the algorithms currently supported, such as SVC, KNN classifier, and DBSCAN. Nowadays, I am looking for some kind of optimized and fast implementation of RadiusNeighborsClassifier. It would be good to have that implementation in this library.
Thank you very much for your work.

**Comment by syakov-intel** [comment]
Currently we do not plan to implement this algorithm without enough evidence of its broad usage. Although contribution would be welcome!
